src/FastRouteRouter.php

```diff
@@ -80,15 +109,31 @@ class FastRouteRouter implements RouterInterface
      *     implementation will be used.
      * @param null|callable $dispatcherFactory Callable that will return a
      *     FastRoute dispatcher.
+     * @param array $config Array of custom configuration options.
      */
-    public function __construct(RouteCollector $router = null, callable $dispatcherFactory = null)
-    {
+    public function __construct(
+        RouteCollector $router = null,
+        callable $dispatcherFactory = null,
+        array $config = null
+    ) {
         if (null === $router) {
             $router = $this->createRouter();
         }
 
         $this->router = $router;
         $this->dispatcherCallback = $dispatcherFactory;
+
+        if (isset($config['cache_enabled'])) {
```

Comment:
@Ocramius 
done here: https://github.com/zendframework/zend-expressive-fastroute/pull/16/commits/b51205518de3aae08a51b27cf3fe4c6925f22a4f

I am not sure about the call to loadDispatchData; maybe we could keep only the reading of $config keys in there (for other future extra parameters) and move back that call into the constructor after the loadConfig call. Which option do you think best?

kind regards